nuitka/MainControl.py

```diff
@@ -24,9 +24,12 @@
 
 """
 
+import cProfile
 import os
 import sys
 
+import pyprof2calltree
```

Comment:
These imports should be hidden away where they are used, maybe even its own module is warranted, this slows down Nuitka startup for all users, and makes these modules mandatory.